File: c/2357.c
int minimumOperations(int* n, int l) {
    bool f[101] = {false};
    int c = 0;
    for (int i = 0; i < l; i++) {
        if (n[i] && !f[n[i]]) {
            f[n[i]] = true;
            c++;
        }
    }
    return c;
}

File: javascript/2357.js
/**
 * @param {number[]} n
 * @return {number}
 */
var minimumOperations = function (n) {
    let f = new Array(101).fill(false);
    let c = 0;
    for (let i of n) {
        if (i != 0 && !f[i]) {
            f[i] = true;
            c++;
        }
    }
    return c;
};
